src/main/java/org/team4159/frc/robot/Constants.java

```diff
@@ -55,5 +61,32 @@
   public final static class CONTROLS {
     public static final int LEFT_JOY = 0;
     public static final int RIGHT_JOY = 1;
+    public static final int SECONDARY_JOY = 2;
+  }
+
+  public final static class SHOOTER_CONSTANTS {
+    public static final double kP = 0.001;
+    public static final double kI = 0.0;
+    public static final double kD = 0.0;
+  }
+
+  public final static class ARM_CONSTANTS {
+    public static final int LIMIT_SWITCH_PORT = 9;
+
+    public static final int ENCODER_CHANNEL_A_PORT = 0;
+    public static final int ENCODER_CHANNEL_B_PORT = 1;
+    public static final boolean IS_ENCODER_REVERSED = true;
+    public static final EncodingType ENCODER_ENCODING_TYPE = EncodingType.k4X;
+
+    public static final double kP = 0.05;
+    public static final double kI = 0.0;
+    public static final double kD = 0.00;
+
+    // TODO: Find
+    public static final int UP_POSITION = 0;
```

Comment:
up_position for what (I'm assuming arm)? Should specify what it's for and what unit the number is representing 

ARM_UP_POSITION_DEGREES